SKILL.tsv

```diff
@@ -5587,7 +5587,7 @@ SKILL_20151102_005586	$Increased physical attack and magic attack
 SKILL_20151102_005587	$Gae Bulg: Defense Zone
 SKILL_20151102_005588	$Increased physical defense and magic defense
 SKILL_20151102_005589	$Increases block penetration by 10% per attribute level when attacking an enemy with a [Two-handed Spear].
-SKILL_20151102_005590
+SKILL_20151102_005590	$[Gae Bulg] causes [Bleeding] in targets hit. [Bleeding] duration increases by 1 second per level, and damage is proportional to the character's physical attack.
```

Comment:
"Afflicts [Bleeding] on an enemy attacked by [Keel Hauling] for 5 seconds. Increases the duration of [Bleeding] by 1 second per attribute level. Bleeding damage is proportional to the character's STR."